src/com/gh4a/db/DbHelper.java

```diff
@@ -22,6 +23,12 @@ public void onCreate(SQLiteDatabase db) {
                 + "type integer not null, "
                 + "uri text not null, "
                 + "extra_data text);");
+        db.execSQL("create table " + SUGGESTIONS_TABLE + " ("
+                + "_id integer primary key autoincrement, "
+                + "type integer not null, "
+                + "suggestion text, "
+                + "date long, "
+                + "unique (type, suggestion) on conflict replace);");
```

Comment:
This also needs to be done in onUpgrade() (and the DB version also needs to be bumped). It's probably worth being put in a separate method, which is then called from both places.